packages/react-aria-components/src/Checkbox.tsx

```diff
@@ -182,12 +182,8 @@ function Checkbox(props: CheckboxProps, ref: ForwardedRef<HTMLLabelElement>) {
     // eslint-disable-next-line react-hooks/rules-of-hooks
     }, useToggleState(props), inputRef);
   let {isFocused, isFocusVisible, focusProps} = useFocusRing();
-  let isInteractionDisabled = isDisabled || isReadOnly;
 
-  let {hoverProps, isHovered} = useHover({
-    ...props,
-    isDisabled: isInteractionDisabled
-  });
+  let {hoverProps, isHovered} = useHover(props);
```

Comment:
hmm interesting. Should read only have hover states? Seems weird?

Reply:
https://github.com/adobe/react-spectrum/pull/5559#discussion_r1425846787